### PR TITLE
plot_wavebin_series interactive plot labels

### PR DESCRIPTION
**Pull request by kglidic**

### CODE REVIEW

File: tshirt/pipeline/spec_pipeline.py [hunk 1]
```diff
@@ -1345,6 +1345,8 @@ def plot_wavebin_series(self,nbins=10,offset=0.005,savePlot=True,yLim=None,xLim=
             p = bokeh.plotting.figure()
             p.background_fill_color="#f5f5f5"
             p.grid.grid_line_color="white"
+            p.xaxis.axis_label = 'Time (JD - {})'.format(offset_time)'
```
Comment (eas342) [inline tshirt/pipeline/spec_pipeline.py]: Has an extra quote mark at the end

**Review by eas342 — COMMENTED** [review]
Check syntax - quote mark

**Comment by eas342** [comment]
Excellent update, thanks!

**Comment by kglidic** [comment]
Thanks! I think I'm slowly getting the hang of working on GitHub.

**Pull request merged**